pymepix/processing/acquisition.py

```diff
@@ -133,126 +65,25 @@ def __init__(self, data_queue, address, longtime):
             self, data_queue, address, longtime, use_event=True, name="Centroid"
         )
         self.info("Initializing Centroid pipeline")
-        self._skip_centroid = 1
-        self._tot_threshold = 0
-        self._samples = 5
-        self._epsilon = 2.0
+        self.centroid_calculator=CentroidCalculator()
 
-        self.addStage(4, Centroiding, num_processes=25)
+        self.addStage(4, PipelineCentroidCalculator, num_processes=6)
```

Comment:
This works only in pymepix-viewer currently. The property numBlobProcesses can be changed. Before it is changed the pipeline is stopped (in the viewer) and started again afterward. Otherwise its not possible to change the number of Processes. In this case the 6 is only the initial value can not be changed from the call of the constructor. This is also not intended by design of the pipeline as the instance is created in some kind of hidden way (Not my implementation that how it already was before).